### ISSUE TITLE
github.com/HubSpot/YouMightNotNeedjQuery

### ISSUE THREAD

**Issue by mintyPT** [oneuijs/You-Dont-Need-jQuery#3]
I don't know if you saw this before https://github.com/HubSpot/YouMightNotNeedjQuery

Maybe it would be a good idea to merge both if there are cases missing from that repo.

**Comment by camsong** [comment]
Yep for sure, [YouMightNotNeedjQuery](https://github.com/HubSpot/YouMightNotNeedjQuery) is another good guide to reach, I also made a [PR](https://github.com/HubSpot/youmightnotneedjquery/pull/110) to that repo, but unfortunately the maintainers seems not so active. That's also why we started this project.

Basically this project is different with YouMightNotNeedjQuery in these:
* Every piece of tips in this guide is used in our projects on PRODUCTION, so feel free to copy them.
* Currently we only focused on IE 10+, for it's more featureful. In the near future we don't have enough energy to work with IE 8/9
* This project will be actively maintained.
